src/SharedUtilities/models.jl

```diff
@@ -96,23 +132,18 @@ have different dimensions - require defining a new method.
 """
 function initialize_prognostic(
     model::AbstractModel{FT},
-    state::Union{ClimaCore.Fields.Field, Vector{FT}, FT},
```

Comment:
PS just a note - not sure where to store this mentally:

once Point is a CC domain, we dont need to support "state" being a Vector{FT} as well...